jfr-daemon/src/main/java/com/newrelic/jfr/daemon/VersionFinder.java

```diff
@@ -24,23 +25,23 @@ private VersionFinder() {}
    * @return the version string
    */
   public static String getVersion() {
-    var classLoader = VersionFinder.class.getClassLoader();
+    ClassLoader classLoader = VersionFinder.class.getClassLoader();
     if (!(classLoader instanceof URLClassLoader)) {
       return UNKNOWN_VERSION;
     }
-    var cl = (URLClassLoader) classLoader;
-    var url = cl.findResource("META-INF/MANIFEST.MF");
+    URLClassLoader cl = (URLClassLoader) classLoader;
+    URL url = cl.findResource("META-INF/MANIFEST.MF");
```

Comment:
We need to talk about this usage of URLClassLoader

Reply:
Created a [new issue](https://github.com/newrelic/newrelic-jfr-core/issues) to address this.